src/packages/sdk/src/__tests__/__snapshots__/convertCredentials.test.ts.snap

```diff
@@ -126,23 +126,59 @@ Object {
 
 exports[`Convert mongodb://myDBReader:D1fficultP%40ssw0rd@mongodb0.example.com:27017/admin 2`] = `"mongodb://myDBReader:D1fficultP%40ssw0rd@mongodb0.example.com:27017/admin"`;
 
-exports[`Convert mysql://root@/db?socket=/private/tmp/mysql.sock 1`] = `
+exports[`Convert mysql://root@localhost/db?socket=/private/tmp/mysql.sock 1`] = `
 Object {
   "database": "db",
   "extraFields": Object {},
-  "host": undefined,
+  "host": "localhost",
   "password": undefined,
   "port": undefined,
   "schema": undefined,
   "socket": "/private/tmp/mysql.sock",
   "ssl": false,
   "type": "mysql",
-  "uri": "mysql://root@/db?socket=/private/tmp/mysql.sock",
+  "uri": "mysql://root@localhost/db?socket=/private/tmp/mysql.sock",
```

Comment:
The downside of this approach is that this URL now errors like

```
> new URL('mysql://root@/db?socket=/private/tmp/mysql.sock')
Uncaught:
TypeError [ERR_INVALID_URL]: Invalid URL: mysql://root@/db?socket=/private/tmp/mysql.sock
    at onParseError (internal/url.js:258:9)
    at new URL (internal/url.js:334:5)
    at REPL19:1:1
    at Script.runInThisContext (vm.js:132:18)
    at REPLServer.defaultEval (repl.js:484:29)
    at bound (domain.js:430:14)
    at REPLServer.runBound [as eval] (domain.js:443:12)
    at REPLServer.onLine (repl.js:817:10)
    at REPLServer.emit (events.js:327:22)
    at REPLServer.EventEmitter.emit (domain.js:486:12) {
  input: 'mysql://root@/db?socket=/private/tmp/mysql.sock',
  code: 'ERR_INVALID_URL'
}
```

Reply:
This error is now caught and replaced with in `uriToCredentials` function 
`Invalid data source URL. See https://www.prisma.io/docs/reference/database-reference/connection-urls`